### PR TITLE
Removed outdated robo watch task

### PR DESCRIPTION
**Pull request by dipakmdhrm**
#324

### CODE REVIEW

File: RoboFile.php [hunk 1]
```diff
@@ -180,16 +181,17 @@ protected function monitoredThemeDirectories(): array {
   public function themeWatch(): void {
     $this->say('Compiling and watching (optimized).');
     $this->doThemeCompile(TRUE);
+    $watch = $this->taskWatch();
     foreach ($this->monitoredThemeDirectories() as $directory) {
-      $this->taskWatch()
```
Comment (dipakmdhrm) [inline RoboFile.php:184]: Reference from here: https://robo.li/tasks/Base/#watch

**Comment by amitaibu** [comment]
Thanks. Can you please give https://github.com/Gizra/drupal-starter#theme-development an update. It's confusing what needs to be executed - we need to highlight the `ddev theme:watch`

**Comment by dipakmdhrm** [comment]
@amitaibu While updating the README, I noticed that `ddev theme:watch` doesn't watch or compile js files or images. Is that intentional?

**Comment by amitaibu** [comment]
> Is that intentional?

It's not. Are you up tp try and fix that? :)

**Comment by dipakmdhrm** [comment]
> It's not. Are you up tp try and fix that? :)

Sure. I'll give it a try

**Comment by dipakmdhrm** [comment]
@amitaibu Robo has (or 'had' after I removed it in last commit) it's own watch tasks, which looks like can't be combined with tailwind's css tasks. _Are we open to use a js bundler (webpack/gulp or others) for js & image build?_

What I plan to do is:
- Keep the existing robo image build task
- Keep the existing postcss css build task
- Create the new js build task
- Use the bundler to combine these three in a compile & a watch task.

**Comment by amitaibu** [comment]
> Are we open to use a js bundler

No, tried a few in the past (Parcel and gulp). They always require more effort than we want. And Robo I believe should give us that (copy, optimize images, optimize JS)

**Comment by dipakmdhrm** [comment]
Closing this for #330

**Pull request closed without merging**